libs/shared/flagd-core/src/lib/flagd-core.ts

```diff
@@ -120,7 +120,7 @@ export class FlagdCore {
     }
 
     const resolvedVariant = flag.variants.get(variant);
-    if (!resolvedVariant) {
+    if (resolvedVariant === undefined) {
```

Comment:
ooof, one of the few times `!someVar` isn't the correct choice :sob:

Reply:
is `null` a valid variant?

Reply:
That should be handled by the type checker that's run immediately after this section.